File: todo/Hello.scala
// Compile using scalac
// Run using scala
object Hello {
    def main(args: Array[String]) = {
        println("Hello from Scala!");
        printf("a + b = %d\n", doAdd(1, 2));
    }

    def doAdd(a: Int, b: Int): Int = {
        return a + b;
    }
}

File: todo/Hello.rs
// Compile using rustc
// Run executable binary
fn do_add(a: i32, b: i32) -> i32 {
    return a + b;
}

fn main() {
    println!("Hello from Rust!");
    println!("a + b = {}", do_add(1, 2));
}
